tests/models/test_backfill.py

```diff
@@ -271,12 +266,12 @@ def _get_bdr(date):
 
     # 2021-01-04 is "failed" so it may or may not be reprocessed depending
     # on the configuration
-    bdr = _get_bdr("2021-01-04")
```

Comment:
changed the date?

Reply:
instead of changing the date, this should probably assert that exception reason is None, and assert that the dag run assorciated with this bdr is the same dag run that was created before the backfill.  to document the behavior that we are reusing dag runs and we don't regard it as any kind of exception in that case.